storage/innobase/handler/i_s.cc

```diff
@@ -6652,7 +6672,7 @@ static int i_s_dict_fill_innodb_tablespaces(
         break;
     }
 
-    ut_free(filepath);
+    ut_free(const_cast<char *>(filepath));
```

Comment:
Sorry, it was a mistake to suggest const then :(

Reply:
No problem